js/examples/newsletter_twitter_threads/demo.mjs

```diff
@@ -1,4 +1,5 @@
 import { openai } from "@ai-sdk/openai";
+import {groq} from '@ai-sdk/groq'
```

Comment:
Consider maintaining consistent quote style in imports. The file uses double quotes for other imports but single quotes here. For consistency, suggest using: `import { groq } from "@ai-sdk/groq";`